tests/cypress/e2e/mcpar/form.cy.js

```diff
@@ -110,7 +110,13 @@ function fillOutPartialMCPAR() {
 const traverseRoutes = (routes) => {
   //iterate over each route
   routes.forEach((route) => {
-    traverseRoute(route);
+    // skip over the ILOS routes as they are behind an LD flag
+    if (
+      !route.path.contains("add-in-lieu-of-services") ||
```

Comment:
Wonder if eventually a helper could be useful. Something like `isBehindLdFlag(route.path)` where the flagged routes could be defined in one place.

Reply:
that's a good point! this is the first time we've had to hide a specific report route, but I think that'd be a great enhancement.